wof/core_1_0.py

```diff
@@ -92,21 +93,21 @@ def get_variable_code(self, varArg):
 
     def create_get_site_response(self, siteArg=None):
 
-        if siteArg == None or siteArg == '':
+        if siteArg is None or siteArg == '':
```

Comment:
Only `None`, `True`, and `False` should be check with `is` or `is not`. The one on the right is not a `builtin` type. Although that `if` clause could be changed to `if siteArr:` but I don't want to make such changes until the code is passing all the tests.